docs/how-to/troubleshoot.md

```
@@ -152,41 +152,46 @@ bc agent start agent-name
 
-## Channel Issues
+## Notification Issues
 
-### Messages Not Delivered
+### Notifications Not Delivered
 
-**Cause**: Target agent not running or channel database issue.
+**Cause**: Target agent not running, adapter disconnected, or subscription missing.
 
 **Solution**:
 ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)